test/facade_converter_test.ts

```diff
@@ -76,11 +76,12 @@ function expectErroneousWithType(str: string) {
 describe('type based translation', () => {
   describe('Dart type substitution', () => {
     it('finds registered substitutions', () => {
-      expectWithTypes('import {Observable} from "angular2/src/facade/async"; var p: Promise<Date>;')
+      expectWithTypes(
+          'import {Observable} from "angular2/src/facade/async"; var p: Promise<Date> = new Promise<DateTime>();')
```

Comment:
it is confusing to import Observable and not use it. That's my fault in the last change here. Can you fix that? :rose:
Just split into two expectations...